pkg/util/validation/utils.go

```diff
@@ -0,0 +1,25 @@
+package validation
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Bool2Int(b bool) int {
+	if b {
+		return 1
+	}
+	return 0
+}
+
```

Comment:
Imho functions below should go into `pkg/core/validators`. Especially that one func (`MakeRequiredFieldErr`) seems similar